### PR TITLE
Metrics Glossary: Accuracy

### PR DESCRIPTION
**Pull request by y27choi**
### Linked issue(s):
[KOL-2542](https://linear.app/kolena/issue/KOL-2542/port-accuracyprecisionrecall-doc-to-docskolenaio)

### What change does this PR introduce and why?
Added accuracy, precision and recall guides

### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

File: docs/assets/images/metrics-precision-recall.png
Comment (gordonhart) [inline docs/assets/images/metrics-precision-recall.png:1]: We need to re-make this graphic using our standard colors and styles.

**Comment by gordonhart** [comment]
I'm going to pare this back to just Accuracy and open for review.

**Comment by gordonhart** [comment]
@y27choi let me know what you think. I replaced the sklearn-style `y_true`/`y_pred` with a table and a match formula:

Screen Shot 2023-06-22 at 4 07 50 PM

**Review by y27choi — COMMENTED** [review]
It looks great, @gordonhart!

**Review by gordonhart — APPROVED** [review]

**Pull request merged**